crypto/zkp_ecdsa.c

```diff
@@ -473,3 +474,69 @@ int zkp_ecdh_multiply(const ecdsa_curve *curve,
 
   return result;
 }
+
+// ECDSA public key tweak
+// tweaked_public_key = public_key + private_tweak * G
+// curve has to be &secp256k1
+// public_key_bytes has 33 bytes
+// private_key_tweak_bytes has 32 bytes
```

Comment:
`private_key_tweak_bytes` is actually called `private_tweak_bytes` in the code below. However, I think we should just call it `tweak_bytes`, because there is nothing inherently private about it. In case of BIP32 maybe, because the chain code is not entirely public information, but there are other applications where it does not need to be secret at all. (The latter would imply making the same change in zkp_ecdsa.h, ecdsa.c, ecdsa.h.)

Reply:
Done in https://github.com/trezor/trezor-firmware/pull/4083/commits/3bf6559a80b875348a162aa998d3d213a59f731a.